src/main/java/com/google/api/codegen/GapicContext.java

```diff
@@ -90,4 +94,17 @@ public CollectionConfig getCollectionConfig(Interface service, String entityName
     }
     return result;
   }
+
+  /**
+   * Returns a list of simple RPC methods.
+   */
+  public List<Method> getSimpleRpcMethods(Interface service) {
```

Comment:
@garrettjonesgoogle I was going with the terminology in [gRPC doc](http://www.grpc.io/docs/tutorials/basic/java.html#simple-rpc), but I see how this can cause confusion. I'll change it to `getNonStreamingMethods`.